<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>schedule</class>
 <widget class="QDialog" name="schedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>代办数据管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>序号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>代办内容</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>截止日期</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>创建时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>完成状态</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_add">
       <property name="text">
        <string>新增代办项</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_change">
       <property name="text">
        <string>更改代办项</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete">
       <property name="text">
        <string>删除代办项</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_export">
     <property name="text">
      <string>导出列表</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
